src/server/api/routers/smartsheet/sheets/index.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+import type { Sheet } from "../smartsheetTypes";
+import { createTRPCRouter, publicProcedure } from "../../../trpc";
+
+const sheetRouter = createTRPCRouter({
+  getSheets: publicProcedure.query(({ ctx }) => {
+    const options = {
+      queryParameters: {
+        include: "attachments",
+        includeAll: true,
+      },
+    };
+    const res: Sheet[] = ctx.smartsheet.sheets.listSheets(options);
+    return res;
+  }),
+  getSheet: publicProcedure.input(z.number()).query(({ ctx, input }) => {
+    const res: Sheet = ctx.smartsheet.sheets.getSheet({ id: input });
```

Comment:
also, did we ever verify that there is actually a difference between getSheet and listSheets?

Reply:
No but I am going to go by what the docs say to use since Smartsheet is weird

Reply:
I also got rid of getSheets since I don't need it. So now we only have getSheet.